components/MentorFormRegistration.js

```diff
@@ -0,0 +1,456 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import jwt_decode from "jwt-decode";
+
+import { useRouter } from "next/router";
+import Overlay from "./Overlay";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+export default function MentorForm() {
```

Comment:
this!

Reply:
whole file!

Reply:
what exactly do I have to do?

Reply:
This issue was already there when I started working on my issue. What do I have to do to change it? Because my issue was only to deal with the scroll to top button.

Reply:
arrey your PR contains it.. remove kro may be update ni h tumhari branch.

Reply:
im going to delete this PR, comment mentorformregistration file and reopen a PR. Just a minute.